Tests.Integration.Presentation.Web/Organizations/V2/OrganizationGridConfigurationAPIV2Test.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Core.DomainModel;
+using Core.DomainModel.Organization;
+using Presentation.Web.Models.API.V1;
+using Presentation.Web.Models.API.V2.Internal.Response.Organizations;
+using Tests.Integration.Presentation.Web.Tools;
+using Tests.Toolkit.Patterns;
+using Xunit;
+
+namespace Tests.Integration.Presentation.Web.Organizations.V2
+{
+    public class OrganizationGridConfigurationApiV2Test : WithAutoFixture
+    {
+
+        [Fact]
+        public async Task LocalAdminCanSaveGridConfig()
+        {
+            var (org, cookie) = await CreatePrerequisites();
+
+            var columns = CreateTestColumns().ToList();
+            var saveResponse = await OrganizationGridConfigurationTestHelper.SendSaveConfigurationRequestAsync(org.Uuid, columns, cookie);
+            Assert.Equal(HttpStatusCode.OK, saveResponse.StatusCode);
```

Comment:
I think you can delete this test since the RegularUserCanGetGridConfig does the same + tests if you get the columns